app/src/main/java/com/squirtles/musicroad/common/CreatedByPickText.kt

```diff
@@ -6,40 +6,54 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.SpanStyle
 import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.style.TextDecoration
 import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.text.withStyle
 import com.squirtles.musicroad.R
 
 @Composable
 fun CreatedBySelfText(
     modifier: Modifier = Modifier,
+    showUnderline: Boolean = false,
     color: Color = MaterialTheme.colorScheme.onSurface,
     style: TextStyle = MaterialTheme.typography.bodyMedium
 ) {
+    val myPickDescription = buildAnnotatedString {
+        withStyle(style = SpanStyle(textDecoration = if (showUnderline) TextDecoration.Underline else null)) {
+            append(stringResource(R.string.pick_created_by_self_1))
+        }
+        append(stringResource(R.string.pick_created_by_self_2))
```

Comment:
```suggestion
        append(" ${stringResource(R.string.pick_created_by_self_2)}")
```

요렇게 바꿔야 할 것 같습니당 `내가등록한`이 붙어있어요!

![image](https://github.com/user-attachments/assets/5f83f765-8d16-487d-8c5d-8045f6ae310b)

반영하면 이렇게 됩니다

![image](https://github.com/user-attachments/assets/93bb34e1-8ac6-4d82-9693-f130b4158596)

Reply:
이거 바꾸면 
![image](https://github.com/user-attachments/assets/1947e149-40af-4813-942c-258d10fafb64)

얘도 바뀌겠죠??

Reply:
네 맞습니다!! 수정하겠습니다